lib/ansible/modules/system/filesystem.py

```diff
@@ -111,20 +119,16 @@ def _get_fs_size(fssize_cmd, dev, module):
                     block_count = int(col[3].split(',')[0])
                     break
         else:
-            module.fail_json(msg="Failed to get block count and block size of %s with %s" % (dev, cmd), rc=rc, err=err )
-    elif 'btrfs' == fssize_cmd:
```

Comment:
Because the current code doesn't support filesystem resizing when the filesystem is `btrfs`. The size in bytes of a `btrfs` filesystem is not 1.